pkgs/by-name/sd/SDL2_image/package.nix

```
@@ -15,6 +15,7 @@
   zlib,
   # Boolean flags
   ## Darwin headless will hang when trying to run the SDL test program
+  enableSTB ? true,
```

```suggestion
  enableSTB ? true,
    ## Darwin headless will hang when trying to run the SDL test program
```
This is confusing ordering